docs/02-app/02-api-reference/05-next-config-js/staleTimes.mdx

```diff
@@ -25,23 +25,22 @@ module.exports = nextConfig
 
 The `static` and `dynamic` properties correspond with the time period (in seconds) based on different types of [link prefetching](/docs/app/api-reference/components/link#prefetch).
 
-- The `dynamic` property is used when the `prefetch` prop on `Link` is left unspecified or is set to `false`.
+- The `dynamic` property is used when the page was not statically generated, nor a "full" prefetch (i.e. with `prefetch={true}`)
```

Comment:
```suggestion
- The `dynamic` property is neither used when the page was not statically generated nor a "full" prefetch (i.e. with `prefetch={true}`)
```

Reply:
Hmm... I don't find that this improves the clarity of the sentence.

Reply:
@ztanner I always use [neither with nor](https://prowritingaid.com/neither-nor-either-or#:~:text=We%20use%20neither%2Fnor%20to%20negate%20each%20of%20two%20things,additional%20nor%20before%20each%20option.) <picture data-single-emoji=":thinkies:" title=":thinkies:"><img class="emoji" width="20" height="auto" src="https://emoji.slack-edge.com/T0CAQ00TU/thinkies/d164ba479fed9c0d.png" alt=":thinkies:" align="absmiddle"></picture>

Reply:
Yeah I think that'd only work if we also get rid of the `not`... ie:

> "The `dynamic` property is used when the page is neither statically generated nor fully prefetched (i.e., with `prefetch={true}`)."

Reply:
Tweaked it to make a bit more sense in [9631dc2](https://github.com/vercel/next.js/pull/68047/commits/9631dc2138bd6999990437fb6592c57f9986119b)